style.css

```diff
@@ -39,15 +52,51 @@
     margin: auto;
     text-align: center
 }
-img[src]{
-    width: 64px;
-    height: 64px;
+.offers-block-header h3{
+    margin: 0;
+    font-weight: 500;
 }
 hr{
-    width: 70px;
+    width: 100px;
+    border: none;
     height: 4px;
-    background: #ffa057;
+    background: #F5BD41;
+}
+.footer{
+    background-color: #10367c;
+    font-family: 'Roboto', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    color: white;
+}
+.footer-inner{
+    display: flex;
+    padding: 120px 0px 65px 100px;
+    min-width: 1200px;
```

Comment:
@Billiardist94 I think it's better to set max-width - as exapmle - set width less than 1200px in responsive in browser - you see -there is a problem =)